crates/sui/src/client_ptb/ast.rs

```diff
@@ -31,6 +31,8 @@ pub const GAS_BUDGET: &str = "gas-budget";
 pub const SUMMARY: &str = "summary";
 pub const GAS_COIN: &str = "gas-coin";
 pub const JSON: &str = "json";
+pub const HELP: &str = "help";
+pub const SHORT_HELP: &str = "h";
```

Comment:
clap usually needs this to be called like `sui client ptb -h`, so not sure if this case will work correctly.